en/setup/dev_env_linux_ubuntu.md

```diff
@@ -1,304 +1,80 @@
 # Development Environment on Ubuntu LTS / Debian Linux
```

Comment:
FYI This now is the sole "ubuntu linux" setup page. It removes most of the explanation of how scripts work, which was a maintenance problem. The only instructions are for using scripts and things that aren't in the scripts.